lib/ramble/ramble/experimental/uploader.py

```diff
@@ -161,6 +161,27 @@ def format_data(data_in):
     return results
 
 
+def results_file_import(filename):
+    """
+    Import Ramble experiment results from a JSON file.
+    """
+    tty.debug("Import file")
+    tty.debug(filename)
+
+    imported_file = open(filename)
+
+    try:
+        tty.msg("Import file...")
```

Comment:
Maybe this is better as a verb, ie `importing`?

Reply:
This is mirroring lines 242 ("Upload experiments...") and 247 ("Upload FOMs..."). I don't have strong feelings for which is better but just wanted to maintain consistency. Shall I update all of them to verbs?

Reply:
Ah yea you're right, thanks for calling that out

I think this is probably fine as is then